jax/_src/lib/xla_bridge.py

```diff
@@ -340,7 +329,10 @@ def backends():
           logging.info("Unable to initialize backend '%s': %s", platform,
                        err)
```

Comment:
Let's reuse this string for both logging and raising an error (i.e. we should say "Unable to initialize backend 'foo':" at the beginning of the error message). You can just raise a RuntimeError with the new message, I don't think we need to preserve the original error type. Also you only need to log in the else case, no need to log if we're about to raise an error.

Reply:
I tried a bunch of things, it seems that the err will be shown anyways when we fall back to CPU and it was not writing the unable to initialize backend unless I make it a warning, then it will make a lot of warnings for TPU, GPU, etc, so I removed the logging altogether. I raise a runtime exception instead of just raising, again because the message was not propagating correctly if I didn't do that! Please let me know what you think.